style.css

```diff
@@ -0,0 +1,36 @@
+    *{
```

Comment:
Remove the extra lines before the css style. Also put a space between the style name and the curly brackets.
In summary each style will look like this:
```
<StyleName> {
...
}
<empty line>
<StyleName2> {
...
}
```